keyboards/trnthsn/s6xty5/keyboard.json

```diff
@@ -0,0 +1,430 @@
+{
+    "manufacturer": "ThanhSon.Mech",
+    "keyboard_name": "S6xty5",
+    "maintainer": "trnthsn",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["B0", "F1", "F0", "C15", "C14", "C13", "A7", "B9", "B8", "B7", "B6", "B5", "B4", "B3", "A15"],
+        "rows": ["B10", "B11", "A2", "A0", "A1"]
+    },
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
```

Comment:
```suggestion
```

Reply:
@fauxpark thank you. I updated my keyboard.json